highLevelInterface/pwaNloptFit.cc

```diff
@@ -183,21 +215,36 @@ rpwa::hli::pwaNloptFit(const pwaLikelihood<complex<double> >& L,
 		TStopwatch timer;
 		timer.Start();
 		nlopt::result result = optimizer.optimize(params, likeli);
-		if(result > 0) {
+		if (result > 0) {
 			converged = true;
 		}
 		timer.Stop();
 
+		// add fixed parameters to params
+		{
```

Comment:
This loop appears 3 times in the code, it would be good to make a function for this.

Reply:
Done. Please check if it is ok this way.